Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/BlueAutonomousCompetition.java

```diff
@@ -0,0 +1,244 @@
+package org.firstinspires.ftc.team6220_2021;
+
+import android.annotation.SuppressLint;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+import org.firstinspires.ftc.robotcore.external.ClassFactory;
+import org.firstinspires.ftc.robotcore.external.hardware.camera.WebcamName;
+import org.firstinspires.ftc.robotcore.external.navigation.VuforiaLocalizer;
+import org.firstinspires.ftc.robotcore.external.tfod.Recognition;
+import org.firstinspires.ftc.robotcore.external.tfod.TFObjectDetector;
+import org.firstinspires.ftc.team6220_2021.ResourceClasses.Constants;
+
+import java.util.List;
+
+@Autonomous(name = "Blue Autonomous Competition", group = "Competition")
+public class BlueAutonomousCompetition extends MasterAutonomous {
+    private static final String TFOD_MODEL_ASSET = "model_20211128_184150.tflite";
+    private static final String[] LABELS = {"TSE"};
+    private static final String VUFORIA_KEY = "AXDNhID/////AAABmTzx9+zSP0cgsSvEBLeS2Y9I1y9lY1nEbJ" +
+            "0/cUmIw6GzDXvrllKLQizl4X4T6iAxXFMJXR5zS8fcXuy6uS6lzlZJOBRnDXn3FusCpuunkIRPgIVyq+peMi" +
+            "d0PN1gwSloq8A+nrV6W1LU10WzZ/Pez2F0to+5aV0bOBB+VhZIdN5ABnoSMPa6JxtR6QaCI3dg++wpGw+/X3" +
+            "RwDJhllOoGVmsLE9DTEuBBAI+MtRIpFNrSR7mcv3TEHMf8YIc+qxED8YE7Az3PGK1xy/NzLqNtFdnNVFhp02" +
+            "38Kaaqnu3DABLRXRjSJ1QRSHmE8mIur5Dk3OcqMv3fwTNt5CnhC2J/D5biVGixUQ+dveylNEVNmp0k";
```

Comment:
Consider putting some of these constants that are used in multiple files in the MasterAutonomous base class. Then they can be used in all the files without being copy pasted.